docs/inc-example-test.md

```diff
@@ -9,10 +9,15 @@
         make run-tests-local
         ```
 
-    - To run a specific test or tests, use the `RUN` variable to pass a regular expression that matches the names of the tests that you want to run:
+    - To run a specific test or tests, use the `RUN` variable to pass a regular expression that matches the names of the tests that you want to run. If you want to match a test name exactly, enclose it with ^ at the start and $ at the end.
 
         ```bash
+        # This will match any test with "TestRunDefaultExample" in its name, 
+        # so it may also run tests like "TestRunDefaultExampleWithFlavor".
         make run-tests-local RUN=TestRunDefaultExample
+
+        # This will only run "TestRunDefaultExample" and exclude any similarly named tests.
+        make run-tests-local RUN="^TestRunDefaultExample$"
         ```
```

Comment:
@rajatagarwal-ibm thanks for this - can you maybe split the 2 example into separate code snippet sections? You can maybe include the descriptions outside of the code snippets then